### ISSUE TITLE
Grammar fixes

### ISSUE THREAD

**Issue by TechnologyClassroom** [athityakumar/colorls#147]
I found a few quick grammar fixes.

Please usually has a comma after it.

Ideologically is mispelled.

- Relevant Issues : (none)
- Relevant PRs : (none)
- Type of issue :
  - [ ] Installation
  - [ ] Font-related
  - [ ] Feature request
  - [ ] Bug in existing feature
  - [X] Developer mode : Code quality / Tests / Documentation

**Comment by athityakumar** [comment]
Thanks for opening this issue, @TechnologyClassroom. I went through [this reference material](https://english.stackexchange.com/questions/389/in-what-cases-should-i-use-a-comma-after-please) regarding whether this change is required or not. To quote from the website,

> For instance, if you had a visitor at your house, who was going to leave the house after you had left it, you might say:
> 
> "Please don't forget to lock the door when you leave."

> However, if this particular person had a habit of leaving the door unlocked, you would want to emphasize the "please", and could do it like so:
>  
> "Please, don't forget to lock the door when you leave."

From our use-case, the former seems to be a better choice (for me) over the latter (unless we judge the user of having a habit of not trying what's written in the README 😉 )

What say?

**Comment by TechnologyClassroom** [comment]
I figured it would be the second case.  This many pleases indicated to me that there were habitually people that did the opposite or nothing at all.  If this is not the case, than the former is appropriate.  Regardless, many of these instances of please could be removed altogether as there are currently seven on the readme.

**Comment by TechnologyClassroom** [comment]
Also user phord goes into greater detail below in that post.  Although the comment has less upvotes, it is a more recent and thoughtful post that may not have had as many eyes available for voting.

**Comment by athityakumar** [comment]
> Regardless, many of these instances of please could be removed altogether as there are currently seven on the readme.

Yup, this was intended. Having more "please"s with lesser emphasis (like, "Please try this solution if you face this issue.") seemed more polite than having lesser number of "please"s that've more emphasis (like, "Please, try this solution if you face this issue.")

**Comment by athityakumar** [comment]
@TechnologyClassroom - Cool, thanks. And sorry for not being able to accept the changes proposed in PR #150. This is why I had added the following to `CONTRIBUTING.md`, right at the beginning. 😞 

>Before sending a Pull Request, please make sure that you're assigned the task on a GitHub issue.
>
> 1. If a relevant issue already exists, discuss on the issue and get it assigned to yourself on GitHub.
> 2. If no relevant issue exists, open a new issue and get it assigned to yourself on GitHub.
>
> Please proceed with a Pull Request only after you're assigned. It'd be sad if your Pull Request (and your hardwork) isn't accepted just because it isn't ideologically compatible.
